### ISSUE TITLE
Email messages themselves read French content in anglicized way by screenreader [XL]

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#942]
In a bilingual Email Notification, Reading all the notification content, English and French in English tone.

Is there a way to identify the language element visibly for English and French content? (even saying 'le français suit')

User would have to say which language -- EN/FR only

https://docs.google.com/document/d/1JaJzEnQ7M6tQRziwEC1q8_UUQBizAdVRtUashXdYZeo/edit

Explore more how to make emails accessible

**Comment by willeybryan** [comment]
Unable to replicate

**Comment by willeybryan** [comment]
Broken out Stage 1 to #1001
